src/Libraries/PythonNodeModels/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +21,5 @@
 [assembly: InternalsVisibleTo("PythonNodeModelsWpf")]
 [assembly: InternalsVisibleTo("PythonMigrationViewExtension")]
 [assembly: InternalsVisibleTo("DynamoCoreWpf")]
+[assembly: TypeForwardedTo(typeof(PythonNodeModels.PythonEngineVersion))]
```

Comment:
Had to move PythonEngineVersion to a lower DynamoServices so that it can be easily accessed in the rest of the projects (without bringing in the PythonNodeModel's dependencies)